src/ngraph/frontend/onnx_import/utils/reshape.cpp

```diff
@@ -24,6 +24,37 @@ namespace ngraph
 {
     namespace onnx_import
     {
+        namespace utils
+        {
+            std::shared_ptr<ngraph::Node> flatten(const std::shared_ptr<ngraph::Node>& node,
+                                                  int axis)
+            {
+                auto data_shape = node->get_shape();
+
+                size_t first_dim = 1;
+                size_t last_dim = 1;
+
+                for (auto index = 0; index < data_shape.size(); ++index)
+                {
+                    last_dim *= data_shape.at(index);
+                    if (index < axis)
+                    {
+                        first_dim = last_dim;
+                    }
+                }
+
+                last_dim /= first_dim;
+
+                std::vector<size_t> axis_order(data_shape.size());
```

Comment:
Please add a line of comment to describe what you want `axis_order` to be.

Reply:
Yes, but what is your intention here? What are you making? A monotonically increasing sequence? The new order of axes? What are you creating? Add a comment about that.